src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Core/EnumerableExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Core
+{
+    /// <summary>
+    ///     Provides extension methods for <see cref="IEnumerable{T}"/>.
+    /// </summary>
+    internal static class EnumerableExtensions
```

Comment:
Wonder if this should go into SDK?

Reply:
It should. But for now we want to put everything in PluginsSystem folder. We can do a refactor once plugins stuff goes official.

Reply:
Same for the other comment :)